### ISSUE TITLE
Greek modes?

### ISSUE THREAD

**Issue by cortsf** [twitchax/kord#20]
It's possible to show greek modes for a given root? To achieve the equivalent of this (using [pedrozath/coltrane](https://github.com/pedrozath/coltrane))

```
$ coltrane show scale scale:Dorian tone:C representation:Text
C D E♭ F G A B♭
```

The `--help` messages of kord says:

```
* Any modifiers (e.g., `7`, `9`, `m7b5`, `sus4`, `dim`, `+`, `maj7`, `-maj7`, `m7b5#9`, etc.).
```

Not sure if "etc" includes any notation to specify modes.

**Comment by twitchax** [comment]
I'm not sure I fully understand, but you're saying you would like to be able to type `C Dorian`, and get that output?

**Comment by cortsf** [comment]
Yep.  If this is not suported and a feature request could be accepted, numbers (or roman numerals) are shorter, faster to type and nice to have as an alternative to the greek names. For example `C_2 == `C_Dorian`, `C_5` == `C_Mixolydian`, etc.

**Comment by twitchax** [comment]
Yeah, cool.  That would be fairly trivial to implement.

If you're interested, you are welcome to contribute.  Otherwise, I intend to rework the web experience for this library "soon", and I can make this enhancement at the same time.

**Comment by cortsf** [comment]
Ok, I'll look the code but I can't promess to submit a PR. Not very familiar with rust.

P.S: Thanks for making kord  😃

**Comment by twitchax** [comment]
No pressure at all.

Do you have any deadlines.  I may be convinced to do this earlier, lol.

**Comment by cortsf** [comment]
I'll probably look at the code tomorrow but most likely I won't do it at least in the short term,  if it's not very obvious how it works and how to do it.

**Comment by twitchax** [comment]
Coming in 0.8.0.
